app/user/profile/LogoutButton.tsx

```diff
@@ -14,16 +14,17 @@ export const LogoutButtonSkeleton = () => (
   />
 )
 
-export type LogoutButtonProps = {
-  className?: string
+type LogoutButtonProps = {
+  onClick?: () => void
 }
+const className = "ml-auto lg:order-2 min-w-40"
+const size = "sm"
 
-const LogoutButton = ({ className }: LogoutButtonProps) => {
-  const [logoutWasClicked, setLogoutWasClicked] = useState(false)
-  const { session, isLoading } = useSession()
+const LogoutButton = ({ onClick }: LogoutButtonProps) => {
+  const { session, isLoading: sessionIsLoading } = useSession()
```

Comment:
We should handle this on the user profile page, so we never actually see this button and have to make this check here.